app/src/ui/mapview/Mapview.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import clsx from "clsx";
+import styles from "./Mapview.module.scss";
+
+import { MapViewProps } from "./Mapview.types";
+import { MapViewProvider } from "hooks/useMapContext/useMapContext";
+
+export const MapView: React.FC<MapViewProps> = ({ children, className, mapOptions }) => {
```

Comment:
The file is `Mapview` but the component is `MapView`
Let's rename the file to `MapView`.

Remember to run `git rm --cached -r .` for the rename to take effect.

Or better: https://www.patrick-wied.at/blog/rename-files-and-folders-with-git